components/kyma-environment-broker/cmd/broker/suite_test.go

```diff
@@ -927,7 +927,7 @@ func fixConfig() *Config {
 			DeprovisioningTimeout: 2 * time.Minute,
 		},
 		Reconciler: reconciler.Config{
-			ProvisioningTimeout: 5 * time.Second,
+			ProvisioningTimeout: 2 * time.Minute,
```

Comment:
will remove this after https://github.com/kyma-project/control-plane/pull/1748 merges